Add plugin: Instant Above Divider

sedationh opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/sedationh/obsidian-instant-above-divider

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
Remove the `styles.css` file from your release, since it's empty it's just unnecessary bloat.

[# Obsidian Instant Above Divider](https://github.com/sedationh/obsidian-instant-above-divider/blob/b50587900d3ddb2b29ff2bb99184f45b880e9ede/README.md#L1)
Naming something "Obsidian xyz" is reserved for first party products we create, please change this header.

[await this.loadSettings();](https://github.com/sedationh/obsidian-instant-above-divider/blob/b50587900d3ddb2b29ff2bb99184f45b880e9ede/main.ts#L14)
Your plugin does hot have any settings, so remove this function call.

sedationh commented:
@joethei 

1. removed styles.css
2. removed Obsidian text from title https://github.com/sedationh/obsidian-instant-above-divider?tab=readme-ov-file#instant-above-divider
3. removed setting relevant code

1.0.8 released https://github.com/sedationh/obsidian-instant-above-divider/releases/tag/1.0.8
